<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>FormatDialog</class>
    <widget class="QDialog" name="FormatDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>480</width>
                <height>360</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Dialog</string>
        </property>
        <property name="styleSheet">
            <string notr="true">QDialog{
                border-color: rgb(239, 41, 241);
                border-width: 5px;
                border-style: solid;
                }
            </string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
            <item>
                <widget class="QTextEdit" name="info_edit">
                    <property name="sizePolicy">
                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                        </sizepolicy>
                    </property>
                    <property name="inputMethodHints">
                        <set>Qt::ImhNone</set>
                    </property>
                    <property name="readOnly">
                        <bool>true</bool>
                    </property>
                </widget>
            </item>
            <item>
                <layout class="QGridLayout" name="gridLayout">
                    <item row="0" column="0" colspan="2">
                        <widget class="QToolButton" name="copy_btn">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="styleSheet">
                                <string notr="true">background-color: rgb(143, 89, 2);
                                    color: rgb(255, 255, 255);
                                </string>
                            </property>
                            <property name="text">
                                <string>copy to clip board</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="0">
                        <widget class="QToolButton" name="save_btn">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="styleSheet">
                                <string notr="true">background-color: rgb(32, 74, 135);
                                    color: rgb(255, 255, 255);
                                </string>
                            </property>
                            <property name="text">
                                <string>save</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="1">
                        <widget class="QToolButton" name="back_btn">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="styleSheet">
                                <string notr="true">background-color: rgb(196, 160, 0);
                                    color: rgb(255, 255, 255);
                                    border-color: rgb(39, 41, 41);
                                    border-width: 5px;
                                    border-style: solid;
                                </string>
                            </property>
                            <property name="text">
                                <string>back</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
